docs/File-Formats.md

```diff
@@ -677,6 +677,18 @@ The `cbp_driver` column flags the mutation as either driver or passenger. In cBi
 You can learn more about configuring these annotations in the [portal.properties documentation](portal.properties-Reference.md#custom-annotation). When properly configured, the customized annotations appear in the "Mutation Color" menu of the OncoPrint view: \
 ![schreenshot mutation color menu](images/screenshot-mutation-color-menu.png) 
 
+#### Allele specific copy number (ASCN) annotations
+Allele specific copy number (ASCN) annotation is also supported and may be added using namespaces, described [here](#adding-mutation-annotation-columns-through-namespaces). If ASCN data is present in the MAF, the deployed cBioPortal instance will display additional columns in the mutation table showing ASCN data. All of the following columns must be present for ASCN data to show up in portal.
```

Comment:
I think that this should be mentioned after the [Addition mutation annotation columns through namespaces](https://github.com/mandawilson/cbioportal/pull/6/files#diff-fc80633ba06596c0353416ca1e72a3caR695) since this section introduces what namespaces are and how you can add them to your MAF. ASCN support is just an applied version of `namespaces`